CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when partitions are required. Note that, this feature is available for preprocessing/partitions but is under development for split based algorithms (`split_solve` and `dichotomous`)
+- Position field for activities
+- Sheduling and initial routes are now compatible
+- Unfeasible missions are removed from routes [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Log reasons of unfeasible missions [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Read and log OR-Tools version
+- Route costs [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Log of matrix computation [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- [WIP] Allow multidepot within clustering [#56](https://github.com/Mapotempo/optimizer-api/pull/56)
+- Introduce direct shipments, pickup and delivery must be consecutive [#51](https://github.com/Mapotempo/optimizer-api/pull/51)
+- Return `total_time` and `total_travel_distance` from scheduling heuristic [#63](https://github.com/Mapotempo/optimizer-api/pull/63)
+- Return original ids for missions and vehicles [#73](https://github.com/Mapotempo/optimizer-api/pull/73)
+- Return `total_waiting_time` [#83](https://github.com/Mapotempo/optimizer-api/pull/83)
+- Introduce `minimize_days_worked` parameter for scheduling heuristic [#89](https://github.com/Mapotempo/optimizer-api/pull/89)
```

Comment:
Reword : - Change default behavior of scheduling heuristic to balance planned services over the entire period and introduce `minimize_days_worked` parameter for previous behavior